database_seed.py

```diff
@@ -1,51 +1,361 @@
 from eagleEvents.models import Company, Customer, Event, Table, Guest, SeatingPreferenceTable, SeatingPreference, User
 from eagleEvents import db
 import datetime
+from random import randint
+import random
+random.seed()
+
+'''
+Private helper functions for the main seed function. 
+'''
+
+def __get_company_name(i: int):
+    """
+    used to get name for each of our 10 test customers
+        :param i:int: which company to get
+    """
```

Comment:
This works but another way to do this would be to create a dictionary or list and just use the int to index into it instead of all the ifs and just check if the int is bigger than the list to handle your else case

Reply:
This comment is about the lines below this comment